### PR TITLE
Some apps don't have a process_command_send_message()

### PR DESCRIPTION
**Pull request by jerith**
Keeping this ticket as a collection of those as I'm not sure how to go about this with the new routing setup.

Currently this is:
- [ ] jsbox (http://prd-sentry.za.prk-host.net/vumi/vumi-go-prod/group/1815/)

### CODE REVIEW

File: go/vumitools/app_worker.py [hunk 1]
```diff
@@ -380,12 +380,36 @@ def process_command_stop(self, user_account_key, conversation_key):
...
-    def process_command_initial_action_hack(self, *args, **kwargs):
-        # HACK: This lets us do whatever we used to do when we got a `start'
-        # message without having horrible app-specific view logic.
-        # TODO: Remove this when we've decoupled the various conversation
-        # actions from the lifecycle.
-        pass
+    @inlineCallbacks
+    def process_command_send_message(self, user_account_key, conversation_key,
+                                     **kwargs):
+        command_data = kwargs.pop('command_data')
```
Comment (hodgestar) [inline go/vumitools/app_worker.py]: Can we just include command_data in the function signature?

**Comment by hodgestar** [comment]
:+1:

File: go/vumitools/app_worker.py (hunk 1 above)
Comment (jerith) [inline go/vumitools/app_worker.py]: Good point.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**